### PR TITLE
[Docs] split api customization docs

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         | 1.10
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: docs/customization/api/modifying_endpoints.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+How to rename an endpoint's path?
+---------------------------------
+
+If you want to change an endpoint's path, you just need to change the ``path`` attribute in your config:
```
Comment (lchrusciel) [inline docs/customization/api/modifying_endpoints.rst:4]: We should probably mention, that in order to do that, you have to duplicate whole config in the first place

**Review by lchrusciel — APPROVED** [review]

File: docs/customization/api/modifying_endpoints.rst (hunk 1 above)
Comment (SirDomin) [reply docs/customization/api/modifying_endpoints.rst:4]: thats not part of the task tho

**Review by AdamKasp — APPROVED** [review]

**Comment by AdamKasp** [comment]
Thank you, @SirDomin! :1st_place_medal:

File: docs/customization/api/modifying_endpoints.rst (hunk 1 above)
Comment (lchrusciel) [reply docs/customization/api/modifying_endpoints.rst:4]: not entirely true, as this issue does not existed in the previous version, as everything was in one doc

**Pull request merged**